Jenkinsfile

```diff
@@ -36,16 +36,21 @@ pipeline {
         axes {
           axis {
             name 'NODE_VERSION'
-            values '10', '12', '14'
+            values '14', '16'
           }
         }
 
         agent {
-          docker {
-            image "us.gcr.io/logdna-k8s/node:${NODE_VERSION}-ci"
+          node {
+            label 'ec2-fleet'
+            customWorkspace "${PROJECT_NAME}-${BUILD_NUMBER}"
           }
         }
 
+        tools {
+          nodejs "NodeJS ${NODE_VERSION}"
```

Comment:
I’d do this in docker still so we get the latest patch version for each

Reply:
What's the risk of not being at the current patch version.  As long as the ones we're running are stable, that should be good enough since we'll never be able to always be current.  I feel docker is unnecessary overhead for a standalone package like this, no?

Reply:
The commit has been dropped to keep this moving along.